lms/templates/course_modes/choose.html

```diff
@@ -12,6 +12,257 @@
     ${_("Enroll In {course_name} | Choose Your Track").format(course_name=course_name)}
 </%block>
 
+<%block name="testcss">
+<style>
+.v2.register-choice {
```

Comment:
i was concerned about putting test quality css that we would probably want to improve if we decided to implement this in a regular css file, perhaps we could create a css file for tests in edx-platform like we did for edx-mktg? although not entirely sure how that works in the platform